### PR TITLE
update 1.8.1

### PR DESCRIPTION
**Pull request by Arishamays1**
##  Dacite 1.8.1 Update
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2283)
[Upstream](https://github.com/konradhalas/dacite)
[Dependencies ](https://github.com/konradhalas/dacite/blob/master/setup.py)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies
- Additional information in `about` section

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True # [py<37]
+  script: {{ PYTHON }} -m pip install .  --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - python
     - dataclasses
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

~https://anaconda.org/anaconda/dataclasses~

see comment from @skupr-anaconda

Comment (skupr-anaconda) [reply recipe/meta.yaml]: I would remove it from `host`. It's a runtime dependency for py<37

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
+    - setuptools
+    - wheel
   run:
     - python
     - dataclasses
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - dataclasses  # [py<37]
```
see https://github.com/konradhalas/dacite/blob/10a9ec40fc5874ae434aa68b975d1b1bf667a42f/setup.py#L31

Comment (Arishamays1) [reply recipe/meta.yaml]: Got it! Made this amendment. Thanks!

**Review by skupr-anaconda — APPROVED** [review]

**Review by lorepirri — APPROVED** [review]
LGTM!

**Comment by lorepirri** [comment]
Ah just one thing for the next time: please use the ticket number (e.g. `PKG-2283`) in the name of the branch or in the title of the PR to enable the Jira's tracking for this PR!

**Comment by Arishamays1** [comment]
> Ah just one thing for the next time: please use the ticket number (e.g. `PKG-2283`) in the name of the branch or in the title of the PR to enable the Jira's tracking for this PR!

Thanks for letting me know this! This makes sense. Jira seems to be tracking it fine, but this is a good precaution will do!

**Pull request merged**